src/test/java/integration/ClientControllerIntegrationTest.java

```diff
@@ -0,0 +1,125 @@
+package integration;
+
+import java.util.Random;
+
+import io.restassured.response.Response;
+import org.json.simple.JSONObject;
+import org.junit.Test;
+
+import static io.restassured.RestAssured.*;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.greaterThan;
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Created by Diarsid on 22.03.2017.
+ */
+public class ClientControllerIntegrationTest {
+    
+    @Test
+    public void getAllClients() throws Exception {
+        Response response = expect().statusCode(200).when().get("/clients");
+        int status = response.getStatusCode();
+        assertEquals(200, status);
+    }
+    
+    @Test
+    public void getClientById() throws Exception {
+        expect().statusCode(200).when().get("clients/1").then().body("id", equalTo(1));
+        expect().statusCode(200).when().get("clients/2").then().body("id", equalTo(2));
+    }
+    
+    @Test
+    public void getClientById_notFound() throws Exception {
+        expect()
+                .statusCode(404)
+                .when()
+                .get("clients/{id}", new Random().nextInt());
```

Comment:
Isn't there a tiny chance to generate existing id and test will fail in this case? I know, it's super unlikely, but still, it bothers me a bit.